docs/usage/settings.md

```diff
@@ -44,4 +55,22 @@ so in this example `redis_host` could only be modified via `export REDIS_HOST`.
 
 !!! note
     On Windows, python's `os` module always treats environment variables as case-insensitive, so the
-    `case_sensitive` config setting will have no effect -- settings will always be updated ignoring case.
+    `case_sensitive` config setting will have no effect - settings will always be updated ignoring case.
+
+## Parsing environment variable values
```

Comment:
I reorganized this file a little to put similar topics into sections. Let me know if you don't like it.

Reply:
looks great.